SurveyApp/Modules/SurveyDetail/SurveyDetailPresenter.swift

```diff
@@ -36,15 +36,26 @@ extension SurveyDetailPresenter: SurveyDetailViewOutput {
             view?.configure(with: survey)
         }
     }
+    
+    func didPressStartSurvey() {
+        if let survey = survey {
+            interactor.fetchSurveyDetail(id: survey.id)
+        }
```

Comment:
[A short explain](https://www.hackingwithswift.com/quick-start/understanding-swift/when-to-use-guard-let-rather-than-if-let) when to use `guard let` rather than `if let` 👍 
```suggestion
        guard let survey = survey else { return }
        interactor.fetchSurveyDetail(id: survey.id)
```